integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDBOperator.java

```diff
@@ -0,0 +1,740 @@
+
+    logger.info("Assign a unique namespace for FMW domain");
+    assertNotNull(namespaces.get(2), "Namespace is null");
+    fmwDomainNamespace = namespaces.get(2);
+
+    logger.info("Assign a unique namespace for WLS domain");
+    assertNotNull(namespaces.get(2), "Namespace is null");
+    wlsDomainNamespace = namespaces.get(3);
+
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(fmwDomainNamespace);
+    createSecretForBaseImages(wlsDomainNamespace);
+
+    //install Oracle Database Operator
+    assertDoesNotThrow(() -> DbUtils.installDBOperator(dbNamespace), "Failed to install database operator");
+
+    logger.info("Create Oracle DB in namespace: {0} ", dbNamespace);
+    dbUrl = assertDoesNotThrow(() -> DbUtils.createOracleDBUsingOperator(dbName, RCUSYSPASSWORD,
+        dbNamespace, WORK_DIR + "/oracledatabase"));
+
+    logger.info("Create RCU schema with fmwImage: {0}, rcuSchemaPrefix: {1}, dbUrl: {2}, "
+        + " dbNamespace: {3}:", FMWINFRA_IMAGE_TO_USE_IN_SPEC, RCUSCHEMAPREFIX, dbUrl, dbNamespace);
+    assertDoesNotThrow(() -> createRcuSchema(FMWINFRA_IMAGE_TO_USE_IN_SPEC, RCUSCHEMAPREFIX,
+        dbUrl, dbNamespace));
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, fmwDomainNamespace, wlsDomainNamespace);
+  }
+
+  /**
+   * Create a basic FMW model in image domain. Create the FMW domain with introspectVersion After domain is created and
+   * introspector exists restart the operator. Verify the restarted operator can find the existing introspector and wait
+   * for it to complete rather than replacing a new introspector. Verify Pod is ready and service exists for both admin
+   * server and managed servers. Verify EM console is accessible.
+   */
+  @Test
+  @DisplayName("Create FMW Domain model in image")
+  void  testFmwModelInImageWithDbOperator() {
+    // Create the repo secret to pull the image
+    // this secret is used only for non-kind cluster
+    createOcirRepoSecret(fmwDomainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(fmwAminSecretName,
+        fmwDomainNamespace,
+        "weblogic",
+        "welcome1"),
+        String.format("createSecret failed for %s", fmwAminSecretName));
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    assertDoesNotThrow(() -> createSecretWithUsernamePassword(fmwEncryptionSecretName,
+        fmwDomainNamespace,
+        "weblogicenc",
+        "weblogicenc"),
+        String.format("createSecret failed for %s", fmwEncryptionSecretName));
+
+    // create RCU access secret
+    logger.info("Creating RCU access secret: {0}, with prefix: {1}, dbUrl: {2}, schemapassword: {3})",
+        rcuaccessSecretName, RCUSCHEMAPREFIX, RCUSCHEMAPASSWORD, dbUrl);
```

Comment:
Change this line 
rcuaccessSecretName, RCUSCHEMAPREFIX, RCUSCHEMAPASSWORD, dbUrl);

to 
rcuaccessSecretName, RCUSCHEMAPREFIX, dbUrl, RCUSCHEMAPASSWORD);

Reply:
good eyes, fixed.